ui/components/ErrorPage.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import {
+  NoSsr,
+  Typography,
+  makeStyles,
+  Link
+} from "@material-ui/core";
+
+const styles = makeStyles(() => ({
+  root : {
+    backgroundColor : "#fff",
+    padding : "2rem",
+    textAlign : "center",
+    borderRadius : 4,
+    height : "100%",
+  },
+  errorSection : {
+  },
+  message : {
+    fontSize : "3rem",
+    lineHeight : "4rem",
+    marginBottom : "2rem",
+  },
+  errMessage : {
+    fontWeight : "400",
+    fontSize : "1.5rem",
+    color : "gray",
+    fontStyle : "italic",
+    marginTop : "2.5rem",
+  },
+  mesh : {
+    display : "block",
+    margin : "auto",
+    marginTop : "3.125rem",
+    maxWidth : "100%",
+    height : "38vh"
+  },
+  helpMessage : {
+    marginTop : "3rem",
+    color : "rgba(0, 0, 0, 0.87)"
+  },
+}))
+
+const customMessages = [
```

Comment:
Just in case if it got missed, actually in the issue description it was asked to use those snarky messages as we are using on our Layer5 website for the 404 page. So, that's the reason I'm using them in the CustomMessages array.

Reply:
Oh, great then

Reply:
Hey, watch your mouth, @theBeginner86. We don't take kindly to snark 'round these parts. 


*ahem*

In all seriousness, The custom error codes are to be presented in the Notification Center, while an invalid URL can simply return a custom 404 page with a bit of sass slapped on it.